contracts/util/NumbersList.sol

```diff
@@ -0,0 +1,64 @@
+pragma solidity 0.5.17;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+library NumbersList {
+    using SafeMath for uint256;
+
+    uint256 private constant PERCENTAGE_TO_DECIMAL = 10000;
+
+    struct Values {
+        uint256 length;
+        uint256 max;
+        uint256 min;
+        uint256 sum;
+    }
+
+    function addValue(Values storage self, uint256 newValue) internal returns (bool) {
+        if (self.max < newValue) {
+            self.max = newValue;
+        }
+        if (self.min > newValue) {
```

Comment:
I think this should be:
 `if (self.min > newValue || self.length == 0) {`

Currently if you have an empty struct, and you try to add the value 2, then you will end up with the following:
```
Values {
        length = 1
        max = 2
        min = 0
        sum = 2
    }
```
Because the initial value of 0 is less than 2

Reply:
I have added new unit tests and fixed some issues.